server/webapp/WEB-INF/spring-all-servlet.xml

```diff
@@ -38,36 +37,20 @@
   <context:component-scan base-package="com.thoughtworks.go.server.web"/>
   <context:component-scan base-package="com.thoughtworks.go.server.newsecurity.controllers"/>
 
+  <mvc:annotation-driven/>
```

Comment:
I believe this is the default behavior with spring.

Reply:
This is needed to instantiate RequestMappingHandlerMapping. For DefaultAnnotationHandlerMapping we don't need that tag.